.github/workflows/container_description.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: Push README to Docker Hub
+on:
+  push:
+    paths:
+      - "README.md"
+      - ".github/workflows/container_description.yml"
+    branches: [ main, master ]
+
+permissions:
+  contents: read
+
+jobs:
+  PushDockerHubReadme:
+    runs-on: ubuntu-latest
+    name: Push README to Docker Hub
+    if: github.repository_owner == 'prometheus' || github.repository_owner == 'prometheus-community' # Don't run this workflow on forks.
+    steps:
+      - name: git checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Set docker hub repo name
+        run: echo "DOCKER_REPO_NAME=$(make common-docker-repo-name)" >> $GITHUB_ENV
+      - name: Push README to Dockerhub
+        uses: christian-korneck/update-container-description-action@d36005551adeaba9698d8d67a296bd16fa91f8e8 # v1
+        env:
+          DOCKER_USER: ${{ secrets.DOCKER_HUB_LOGIN }}
+          DOCKER_PASS: ${{ secrets.DOCKER_HUB_PASSWORD }}
+        with:
+          destination_container_repo: ${{ env.DOCKER_REPO_NAME }}
+          provider: dockerhub
+          short_description: ${{ env.DOCKER_REPO_NAME }}
+          readme_file: 'README.md'
+
+  PushQuayIoReadme:
+    runs-on: ubuntu-latest
+    name: Push README to quay.io
+    if: github.repository_owner == 'prometheus' || github.repository_owner == 'prometheus-community' # Don't run this workflow on forks.
+    steps:
+      - name: git checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Set quay.io org name
+        run: echo "DOCKER_REPO=$(echo quay.io/${GITHUB_REPOSITORY_OWNER} | tr -d '-')" >> $GITHUB_ENV
+      - name: Set quay.io repo name
+        run: echo "DOCKER_REPO_NAME=$(make common-docker-repo-name)" >> $GITHUB_ENV
```

Comment:
@SuperQ I ping you here since you approved https://github.com/prometheus/prometheus/pull/13768

It seems like `prometheus-community/windows_exporter` is out of sync in terms of some prometheus-community repository defaults. Any common makefiles targets are not available. Is there a reason, that the bot does not sync them to windows_exporter?

Reply:
Yes, it looks like the windows_exporter repo doesn't use the standard `Makfile.common` used in other repos.

Actually, I should fix that call to be just `docker-repo-name` so that a local implementation can be done here.

Reply:
https://github.com/prometheus/prometheus/pull/13789

Reply:
The bot, by default, only syncs files that exist in the target repo. Since `Makefile.common` doesn't exist here, it's not synced. I assumed that because there was a `Dockerfile`, that the common makefile also exists. I can fix that.

Reply:
@SuperQ https://github.com/prometheus/prometheus/pull/13789 seems merged now, Does the bot support a rebase?

Reply:
Yes, closed and deleted the branch. Next run should work.